src/arena.rs

```diff
@@ -60,6 +65,9 @@ impl<T: Sync + Send, const BITARRAY_LEN: usize, const LEN: usize> Default
 const fn check_const_generics<const BITARRAY_LEN: usize, const LEN: usize>() {
     let bits = usize::BITS as usize;
 
+    // Rust 1.57 supports `panic!` in const contexts. And utility.rs:12 `option_result_unwrap_unchecked`
+    // is a feature of Rust 1.58, which means the lib msrv is greater that 1.57. So we can replace the
+    // `const_fn_assert` with `panic!` in const contexts.
```

Comment:
Please submit this and remove the const_fn_assert.

I tried before (probably before 1.60) and somehow it didn't work well, so I gave up.

Now it's time to try that again.